ui/src/utils/api-examples/helpers.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Shared helper functions and constants for API example generators
+ */
+
+/**
+ * Standard placeholder values for consistent examples
+ */
+export const PLACEHOLDERS = {
+  rootUrl: 'https://tc.example.com',
+  clientId: 'your-client-id',
+  accessToken: 'your-access-token',
+  taskId: 'dSlITZ4yQgmvxxAi4A8fHQ',
+  taskGroupId: 'dSlITZ4yQgmvxxAi4A8fHQ',
+  runId: '0',
+  workerGroup: 'my-worker-group',
+  workerId: 'my-worker',
+  workerPoolId: 'my-worker-pool',
+  workerType: 'my-worker-type',
+  provisionerId: 'my-provisioner',
+  artifactName: 'public/build/target.tar.gz',
+  hookGroupId: 'project-taskcluster',
+  hookId: 'smoketest',
+  namespace: 'garbage.test-artifacts',
+  cacheName: 'my-cache',
+  roleId: 'repo:github.com/taskcluster/*',
+  clientScopes: ['queue:create-task:*'],
+  secretName: 'project/taskcluster/test',
+  continuationToken: 'continuationToken',
+  limit: '1000',
+};
+
+/**
+ * Map parameter names to appropriate placeholder values
+ *
+ * @param {string} paramName - Parameter name from API definition
+ * @returns {string} Appropriate placeholder value
+ */
+export function getPlaceholderValue(paramName) {
+  // Direct mappings
+  if (PLACEHOLDERS[paramName]) {
+    return PLACEHOLDERS[paramName];
+  }
+
+  // Pattern-based mappings
+  if (paramName.endsWith('Id')) {
+    // For IDs that aren't in our mapping, use a slugid format
+    return 'dSlITZ4yQgmvxxAi4A8fHQ';
```

Comment:
why not use the above-defined placeholders.taskId here? 

Also, it is possible just to use slugid in UI: https://github.com/taskcluster/taskcluster/blob/043af7ec3780fdbbd83ec5dcb1739e8ef6a565e0/ui/src/utils/slugid.js#L42